events/templates/events/film_program.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+
+{% load wagtailcore_tags static wagtailimages_tags i18n %}
+
+{% block extra_css %}
+    {# Override this in templates to add extra stylesheets #}
+    {#  TODO: need to refactor to split to common styles \ results \ film_program  #}
+    <link href="/static/css/results.css" rel="stylesheet">
+    <style>
+        .frame_column {
+            width: 30%;
+        }
+        .frame_column, .film_column {
+            font-weight: bold;
+        }
+    </style>
+{% endblock %}
+
+{% block body_class %}film-program{% endblock %}
+
+{% block content %}
+<div class="row">
+        <h2 id="films"> {{ self.name }} </h2>
+
+        <div class="program-description">
+            {{ self.description | richtext }}
+        </div>
+
+        {% if self.related_films.all %}
```

Comment:
this might retrieve all the related films twice if wagtail doesn't call `select_related`

we need to add debug toolbar to validate questions like this